src/python/grpcio_tests/tests_aio/unit/channel_test.py

```diff
@@ -25,6 +24,167 @@
 from tests_aio.unit._test_base import AioTestBase
 
 
+class TestAioRpcError(unittest.TestCase):
```

Comment:
Should we split this into a separate file?

Reply:
Wanted to have coherence between unit tests and modules, if you believe that we are about to start having too huge files my suggestion would be, lets create the `_call.py` file and let's move the `Call` and `AioRpcError` logic under that file, and so having also a new `call_test.py` file.

How does it sound for you?